object/types.proto

```diff
@@ -181,6 +185,8 @@ message Header {
     // the same `split_id` value.
     bytes split_id = 6 [json_name = "splitID"];
 
+    // Identifier of the initial part of the object split chain
```

Comment:
required?

Reply:
the field? it gonna be the only thing that saves us from ACL rules based on parent meta info

or you mean mentioning that this field is required. are the other onec required?

Reply:
> or you mean mentioning that this field is required

so, answer is "yes"

imo it’s worth explicitly pointing out that the field is present in all parts except the `INIT` itself (this is understandable, but not obvious)

Reply:
> imo it’s worth explicitly pointing out that the field is present in all parts

agree, added some words